src/main/java/com/github/_1c_syntax/bsl/languageserver/diagnostics/LineLengthDiagnostic.java

```diff
@@ -53,6 +57,7 @@ public class LineLengthDiagnostic extends AbstractDiagnostic {
 
   private static final int MAX_LINE_LENGTH = 120;
   private static final boolean CHECK_METHOD_DESCRIPTION = true;
+  private static final boolean EXCLUDE_TRAILING_COMMENTS = false;
```

Comment:
```suggestion
  private static final boolean EXCLUDE_TRAILING_COMMENTS = false;

```

Reply:
The issue is with empty line. There was an empty line between statics and instance fields, you removed it